File: programers_Lv1/수박수박수박수박수박수.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12922?language=python3
def solution(n):
    s_list = ['수', '박']
    answer = ''
    for i, v in enumerate(range(n)):
        if i % 2 == 0:
            answer+=s_list[0]
        else:
            answer+=s_list[1]
    return answer

print(solution(3))

File: programers_Lv1/수박수박수박수박수박수.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12922?language=javascript
function solution(n) {
  const sArr = ["수", "박"];
  let answer = "";
  Array.from({ length: n }, (_, i) => i).forEach((_, i) => {
    answer += i % 2 === 0 ? sArr[0] : sArr[1];
  });
  return answer;
}

console.log(solution(3));
